jib-core/src/main/java/com/google/cloud/tools/jib/cache/Caches.java

```diff
@@ -16,24 +16,57 @@
 
 package com.google.cloud.tools.jib.cache;
 
+import com.google.common.annotations.VisibleForTesting;
+import java.io.Closeable;
+import java.io.IOException;
+import java.nio.file.Files;
 import java.nio.file.NotDirectoryException;
 import java.nio.file.Path;
+import java.nio.file.Paths;
 
 /**
  * Manages both the base image layers cache and the application image layers cache.
  *
  * <p>In general, the cache for base image layers should be shared between projects, while the cache
  * for the application image layers should be specific to a single project.
  */
-public class Caches {
-
-  private final Cache baseCache;
-  private final Cache applicationCache;
+public class Caches implements Closeable {
 
   /** Initializes a {@link Caches} with directory paths. */
   public static class Initializer {
 
-    private Path baseCacheDirectory;
+    /** The default directory for caching the base image layers, in {@code $HOME/.jib-cache/}. */
+    private static final Path DEFAULT_BASE_CACHE_DIRECTORY =
+        Paths.get(System.getProperty("user.home")).resolve(".jib-cache");
```

Comment:
This might be relevant here : https://github.com/GoogleCloudPlatform/appengine-plugins-core/issues/534

Reply:
Ah okay, I will change this to use that method.

Reply:
Filed a separate PR for that: https://github.com/google/jib/pull/154